tests/tacacs/test_ro_user.py

```diff
@@ -60,10 +60,13 @@ def test_ro_user_allowed_command(localhost, duthosts, rand_one_dut_hostname, cre
             'sudo generate_dump',
             'sudo lldpshow',
             # 'sudo psuutil *',
-            'sudo sonic-installer list',
             # 'sudo sfputil show *',
             'sudo ip netns identify 1',
     ]
+    if "201911" in duthosts[rand_one_dut_hostname].os_version:
```

Comment:
it is not this simple, since dev build will not have this in the os_version. the other way is to check the kernel version. you can find some example. that is not ideal, but it is better.

Reply:
i think you can test both sonic_installer and sonic-installer, if the command does not exists, then it will return other error, which is fine.

Reply:
Got it. I decided to test it assuming that at least one of them should be allowed.